base/shell/cmd/cmd.c

```diff
@@ -429,7 +429,7 @@ Execute(LPTSTR Full, LPTSTR First, LPTSTR Rest, PARSED_COMMAND *Cmd)
                           NULL,
                           NULL,
                           TRUE,
-                          0,   /* CREATE_NEW_PROCESS_GROUP */
+                          0,
```

Comment:
That flag is not needed btw?

Reply:
I don't think so. I think the code was originally designed to use that flag to create a child process in its own process group, then GenerateConsolCtrlEvent() was called to send CTRL-C events to it.  However, MSDN says that GenerateConsolCtrlEvent() can not be used in this way.

Reply:
Ah, found the explanation: https://svn.reactos.org/svn/reactos?view=revision&revision=19022